src/coprocessor/dag/expr/builtin_math.rs

```diff
@@ -378,6 +383,32 @@ mod test {
         }
     }
 
+    #[test]
+    fn test_round() {
+        let tests = vec![
```

Comment:
Can you add cases for floating number?
BTW, since all cases use the same signature, you can omit it.

Reply:
@huachaohuang I don't think it is necessary, there is a `round_float` for floating numbers.